src/sentry/api/bases/sentryapps.py

```diff
@@ -33,6 +33,13 @@ def ensure_scoped_permission(request, allowed_scopes):
     return any(request.access.has_scope(s) for s in set(allowed_scopes))
 
 
+def add_integration_platform_metric_tag(func):
+    def decorator(self, *args, **kwargs):
```

Comment:
Should you use [functools.wraps](https://docs.python.org/2/library/functools.html#functools.wraps) to preserve the function's `__name__` and docstring?

Reply:
Probably. TIL.